client/chain-spec/README.md

```diff
@@ -4,7 +4,7 @@ This crate contains structs and utilities to declare
 a runtime-specific configuration file (a.k.a chain spec).
 
 Basic chain spec type containing all required parameters is
-[`ChainSpec`](./struct.ChainSpec.html). It can be extended with
+[`ChainSpec`](https://docs.rs/sc-chain-spec/latest/sc_chain_spec/struct.ChainSpec.html). It can be extended with
```

Comment:
That breaks links when you generate a documentation for your local copy.

Reply:
Good point. I wonder if the majority of devs build their own local copy though vs browsing the online version.
Maybe we can open another (lower) priority issue for that one ?

Reply:
No. The README isn't part of local `cargo doc`s – and this PR only post-processes the README, it does not change the actual source that `cargo doc` will be generated from. Locally generated `cargo docs` are unaffected by this change.

Re linking to a local cargo docs: it didn't  do that before either `./` was a dead link, it would have to be `target/docs` or something.

Reply:
Good point! Marking as resolved then.